src/partials/lines.html

```diff
@@ -0,0 +1,10 @@
+<section class="lines">
```

Comment:
Це оформлення, тому це не повноцінна секція з контентом, а лише div
```suggestion
<div class="lines">
```

Reply:
++